### PR TITLE
docs: adding JWT documentation #7976

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
References http://github.com/lightbend/kalix/issues/7976

removing incorrect assumptions and references to message sign/validation

### CODE REVIEW

**Review by octonato — APPROVED** [review]
LGTM with some suggestions

File: docs/src/modules/spring/pages/using-jwts.adoc [hunk 1]
```diff
@@ -137,13 +135,11 @@ Kalix’s JWT support is configured by placing annotations on methods in your en
...
- Kalix can check for the presence of a JWT token when you invoke your endpoint and include it in your component's metadata. The metadata for a call is available through the context (actionContext, commandContext) of the component. 
- Once your component accesses the token via the metadata - in an entry with the "Authorization" key - you can send it to the authorization server for validation so that your component can respond accordingly. 
+Kalix can validate the signature of JWT tokens provided in an Authorization header to grant access to your endpoints. Typically, these tokens will be generated in response to an authentication request, e.g. by supplying refresh token, or perhaps a username and password. These tokens may be generated by a third party service such as Auth0. You can find more information about the generating tokens https://supertokens.com/blog/are-you-using-jwts-for-user-sessions-in-the-correct-way[here].
 
 === Bearer token validation
 
-If you want to validate that a bearer token is present on a request, this can be done by annotating you endpoint like:
+If you want to validate the bearer token of a request, you need to annotate you endpoint like:
```
[the remark this answers is not in the document]
Comment (franciscolopezsancho) [reply docs/src/modules/spring/pages/using-jwts.adoc:142]: Wouldn't this be a typo, @octonato ?

[the remark this answers is not in the document]
Comment (octonato) [reply docs/src/modules/spring/pages/using-jwts.adoc:142]: Absolutely. I picked the wrong 'you'.

Here is that I want to suggest. 
```suggestion
If you want to validate the bearer token of a request, you need to annotate your endpoint like:
```

[the remark this answers is not in the document]
Comment (franciscolopezsancho) [reply docs/src/modules/spring/pages/using-jwts.adoc:142]: :) thanks, Renato.

**Pull request merged**